sources/gerritDB/helpers.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+exports.injectExcludedProjectsIntoSql = function (excludedProjects, projectColumnName) {
+    var colName = projectColumnName || "dest_project_name";
+    var sql = "";
+
+    if (!excludedProjects || excludedProjects.length === 0) {
+        return " true "; // don't fail conditions
+    }
+
+    excludedProjects.forEach(function (project, i) {
+        if (i === 0) {
```

Comment:
Creating a where clause with "NOT LIKE '%{Project}%' " can cause higher execution time.  I think we should add exact project names into EXCLUDED_PROJECTS in filter_options and change where clause to "NOT IN  (...)  "